### PR TITLE
Close OpenRCT2#13792: Refactor window to class: Park

### PR DESCRIPTION
**Pull request by Makke94**
Refactor of ParkWindow to make use of the new Window class.

### CODE REVIEW

File: src/openrct2-ui/windows/Window.h [hunk 1]
```diff
@@ -80,6 +80,7 @@ rct_window* window_finances_open();
 rct_window* window_finances_research_open();
 
 // WC_PARK_INFORMATION
+rct_window* window_park_open();
```
Comment (duncanspumpkin) [inline src/openrct2-ui/windows/Window.h:83]: Why is there two of these L85 as well? This isn't even called so is pointless.

Comment (Makke94) [reply src/openrct2-ui/windows/Window.h]: Removed

**Comment by Gymnasiast** [comment]
@Makke94 Could you rebase this, and drop the `this->` part of the member access/method calls? (the latter is for consistency with the rest of the codebase)

**Comment by duncanspumpkin** [comment]
I'm closing this as it has gone stale and there have been a few changes to the window that would make rebasing to be hard to achieve.

**Pull request closed without merging**